python/graphstorm/gconstruct/transform.py

```diff
@@ -874,28 +877,30 @@ def label_name(self):
         """
         return self._label_name
 
-    def data_split(self, num_samples):
+    def data_split(self, ids):
```

Comment:
Can we add a unit test for CustomLabelProcessor?
We only have end2end test now.

Reply:
we have unit test for custom data split.

Reply:
https://github.com/awslabs/graphstorm/blob/main/tests/unit-tests/gconstruct/test_construct_graph.py#L599-L697

Reply:
we have many unit tests for custom data split.